src/main/java/io/lsdconsulting/lsd/distributed/interceptor/persistance/QueueService.kt

```diff
@@ -0,0 +1,41 @@
+package io.lsdconsulting.lsd.distributed.interceptor.persistance
+
+import io.lsdconsulting.lsd.distributed.access.model.InterceptedInteraction
+import io.lsdconsulting.lsd.distributed.access.repository.InterceptedDocumentRepository
+import io.lsdconsulting.lsd.distributed.interceptor.config.log
+import java.util.concurrent.*
+import javax.annotation.PostConstruct
+import javax.annotation.PreDestroy
+
+class QueueService(
```

Comment:
minor thoughts: Just pondering on the name again - do we need to mention anything about queuing? Maybe that's an implementation detail that might change 🤔 . (Also my first reaction was to think of pub/sub queues when I saw the name 😆 ). 

Tricky to think of alternatives though.. would RepositoryService work better? 🤷

Reply:
Yes, I see your point. 

It was a proper in-memory pub/sub queue until yesterday 😄 

`RepositoryService` sounds good to me! Thanks!